lib/srv/desktop/discovery.go

```diff
@@ -194,16 +194,19 @@ func (s *WindowsService) lookupDesktop(ctx context.Context, hostname string) (ad
 		return addrs, nil
 	}
 	if s.dnsResolver == nil {
-		return nil, trace.NewAggregate(err, trace.Errorf("DNS lookup for %v failed and there's no LDAP server to fallback to", hostname))
+		return nil, trace.NewAggregate(err, trace.Errorf("DNS lookup for %q failed and there's no LDAP server to fallback to", hostname))
 	}
-	s.cfg.Log.WithError(err).Debugf("DNS lookup for %v failed, falling back to LDAP server", hostname)
+	s.cfg.Log.WithError(err).Debugf("DNS lookup for %q failed, falling back to LDAP server", hostname)
```

Comment:
Why %q here?

Reply:
Well, prior to this change we would actually do a DNS lookup for `hostname=""`, and it's hard to spot the empty string in log messages.

Not strictly necessary, since this PR also fixes the underlying issue, but doesn't hurt.